### PR TITLE
Schema update

### PR DESCRIPTION
**Pull request by CMClay**
#### Overview
ActiveAgents, activeListings, and activeOffices fields have been added to the vendor schema on api.

#### Tasks
- [x] add the fields and descriptions to the vendor schema
- [x] add subtype to coordinates field retsly/api#1509
- [x] add url field to listings, agents, offices, and openhouses

#### Fields to add
- ContractStatusChangeDate
- ContingentDate
- WithdrawnDate
- WithdrawnDate
- OffMarketDate
- OriginalEntryTimestamp
- LastModified Timestamp (agent)

#### Modify
- listDate => contractDate

#### Deploy
- release retsly-schemas version
- bump package.json version on api
- npm install retsly-schemas on api
- build swagger on api
- build docs on www

### CODE REVIEW

**Comment by CMClay** [comment]
@CalvinScott @thalesfsp Quick review please. I'll make a migration for the name change on /api.

**Comment by CalvinScott** [comment]
:+1:

**Comment by thalesfsp** [comment]
:+1:

File: lib/vendor.js [hunk 1]
```diff
@@ -9,5 +9,8 @@ module.exports = {
   update: {type:'date', desc: desc.update},
   crud: {type:'array', subtype: 'string', desc: desc.crud},
   center: {type:'array', subtype: 'number', desc: desc.center},
-  boundary: {type:'array', subtype: 'array', desc: desc.boundary}
+  boundary: {type:'array', subtype: 'array', desc: desc.boundary},
```
Comment (thalesfsp) [inline lib/vendor.js]: @CalvinScott Check this file, add the spaces between the keys and values too in your PR

**Comment by CMClay** [comment]
@CalvinScott This is the latest. Re-review, then i'll release a new version.

File: lib/application-desc.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/**
+* Descriptions for Applications
+*/
+
+module.exports = {
+  name: 'Application name.',
+  status: 'App status: reviewed, inReview, incomplete ,rejected, or suspended',
```
Comment (CalvinScott) [inline lib/application-desc.js]: Space in ,rejected

File: lib/application-desc.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  domains: 'Application domains.',
+  vendors: 'List of applications and their statuses',
```
Comment (CalvinScott) [inline lib/application-desc.js]: .

File: lib/user-desc.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+/**
+* Descriptions for User
+*/
+
+module.exports = {
+  id: 'Unique retsly id.',
+  firstName: 'User first name',
```
Comment (CalvinScott) [inline lib/user-desc.js]: .

File: lib/user-desc.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+  lastName: 'User last name.',
+  email: 'User email',
```
Comment (CalvinScott) [inline lib/user-desc.js]: .

**Comment by CalvinScott** [comment]
Small syntax inconsistencies, but looks good to me :+1:

File: lib/user-desc.js (hunk 5 above)
Comment (CMClay) [inline lib/user-desc.js]: Good catch.

**Comment by CMClay** [comment]
@CalvinScott You're the proofreading king! Thanks!

**Pull request merged**